package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@microsoft/vscode-file-downloader-api",
-    "version": "1.0.1",
+    "version": "1.0.2",
```

Comment:
💡 Recommendation: Thank you so much for this, lets catch-up and make you full access contributors, seems like you have nice ideas around this extension and including you will be a good one so fyi.

Also, Can we make this change as part of separate release PR? WDYT?

Reply:
Sorry - what do you mean? Do you mean - don't increment the version - or go for 1.1.0 - or align the version with the file-downloader repo?